content/apps/production-ready.md

```diff
@@ -2,54 +2,84 @@
 date: 2015-08-28T10:32:59-04:00
 menu:
   main:
-    parent: advanced
-title: Production Ready Checklist
+    parent: apps
+title: Production Ready Guide
 ---
 
-## Zero-Downtime deploy
+# Your guide to having production ready apps on cloud.gov
 
-Your application should be able to be deployed without generating any downtime.
-Check out the [cf-blue-green](https://github.com/18F/cf-blue-green) script as a
-sample script.
+Read this guide early and often, especially when you’re starting to consider a future project. It explains things you can do for reliable and responsive applications deployed on cloud.gov.
 
-## More than one instance
+## Core best practices
+To build consistent, healthy, production-ready applications on cloud.gov incorporate the following practices into your development workflow from the beginning.
 
+### Configuration as code
+To ensure consistency and reproducibility, you should capture your application configuration in version control.
+
+#### How
+* Write an [application manifest](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html) that defines your application configuration, such as the application name, or what previously created services to use.
+
+### More than one instance
 It is critical that your production application has more than one instance so if
-there any issue with one of the runners your app is on your app continues to work.
+there any issue with one of the platform runners where your app instances are assigned, your app continues to function correctly.
+
+#### How
+* See the [multiple instances]({{< relref "multiple-instances.md" >}}) page.
+
+### Prefer User Provided Services
+Prefer user provided services over environment variables for a secure centralized way to store application credentials and variables. Environment variables are ephemeral to the specific deployment of each application.
```

Comment:
I'd go a step further and say "anything sensitive should be in a user-provided service instance, anything non-sensitive should be in the manifest under [`env:`](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#env-block)."